examples/clipboard/Clipboard.cpp

```diff
@@ -0,0 +1,79 @@
+
```

Comment:
I wouldn't mind having examples that even cover small things, but merging it into some bigger example would be nice, yeah.